.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["2.5", "2.6", "2.7", "3.0", "3.1", ruby-head, jruby-9.2, jruby-9.3, jruby-head]
```

Comment:
support for ruby 2.5 and 2.6 have [ended officially](https://endoflife.date/ruby)...should we remove those versions?

Reply:
At the end of the day that's a maintainer decision.  That said, I think it would make sense.

Also, generally speaking I like to bump the gem's major version if I'm going to remove support for a Ruby version.

Reply:
I think we can remove the unsupported versions and release a new major.